### ISSUE TITLE
D_real increases

### ISSUE THREAD

**Issue by ChristianEschen** [junyanz/pytorch-CycleGAN-and-pix2pix#479]
Hi,

I am facing some odd behavior of the loss during training
What does it mean if the D_real suddenly increases while everything else remain constant and L1 is decreasing?

best regards

**Comment by junyanz** [comment]
It might be OK. Could you share us with your plot?
